discovery/module.go

```diff
@@ -449,14 +462,40 @@ func (m *Module) Search(serviceID string, query map[string]string) ([]SearchResu
 			}
 		}
 
+		// extract registrationParameters from VP
+		registrationParameters := extractParameters(matchingVP)
+
 		result = append(result, SearchResult{
 			Presentation: matchingVP,
 			Fields:       fields,
+			Parameters:   registrationParameters,
 		})
 	}
 	return result, nil
 }
 
+func extractParameters(vp vc.VerifiablePresentation) map[string]interface{} {
+	result := make(map[string]interface{})
+	credentials := vp.VerifiableCredential
+	for _, cred := range credentials {
+		if slices.Contains(cred.Type, credential.DiscoveryRegistrationCredentialTypeV1URI()) {
+			credentialSubject := make([]credential.DiscoveryRegistrationCredentialSubject, 0)
+			err := cred.UnmarshalCredentialSubject(&credentialSubject)
+			if err != nil {
+				log.Logger().WithError(err).Error("Failed to unmarshal Discovery Registration Credential Subject")
```

Comment:
maybe add VP ID, so you can track which one it was. Is also an INFO level message at most, I'd say. Since t's not something you can fix in your own infra (probably issue at another participant).